nerdps1

```diff
@@ -474,10 +485,11 @@ function ps1_prompt {
         } else if (zsh) {
             prompt = "%(!.#.$)"
             pre = "%{\033]0;%m\007"
-            post = "%}${_:$((__ps1_noexec=1, 0)):0}"
+            post = "%}"
```

Comment:
**suggestion (non-blocking):** Since this reverts the change made by the first commit, you could simply drop the first commit.

Reply:
ok, done